src/Api/SecretsManager/Models/Request/ServiceAccountsAccessPoliciesRequestModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bit.Api.Utilities;
+using Bit.Core.SecretsManager.Entities;
+using Bit.Core.SecretsManager.Models.Data;
+
+namespace Bit.Api.SecretsManager.Models.Request;
+
+public class ServiceAccountsAccessPoliciesRequestModel
+{
+    public IEnumerable<AccessPolicyRequest> ProjectServiceAccountsAccessPolicyRequests { get; set; }
+
+    public ProjectServiceAccountsAccessPolicies ToProjectServiceAccountsAccessPolicies(Guid grantedProjectId, Guid organizationId)
+    {
+        var projectServiceAccountsAccessPolicies = ProjectServiceAccountsAccessPolicyRequests?
+            .Select(x => x.ToServiceAccountProjectAccessPolicy(grantedProjectId, organizationId)).ToList();
+        var policies = new List<BaseAccessPolicy>();
+
+        if (projectServiceAccountsAccessPolicies != null)
+        {
+            policies.AddRange(projectServiceAccountsAccessPolicies);
+        }
+
+        AccessPolicyHelpers.CheckForDistinctAccessPolicies(policies);
+
```

Comment:
Can we add another request validation here? 

We should only allow `Can read` and `Can read, write` access policies here.

I would recomend adding a new function to the `AccessPolicyHelpers` since we will want to preform this validation on other request models in the future.